cloud/azure/management/recovery/mode/backupjobsstatus.pm

```diff
@@ -153,11 +155,16 @@ sub manage_selection {
         vault_name => $self->{option_results}->{vault_name},
         resource_group => $self->{option_results}->{resource_group}
     );
+
     foreach my $job (@{$jobs}) {
         next if (defined($self->{option_results}->{filter_name}) && $self->{option_results}->{filter_name} ne ''
             && $job->{properties}->{entityFriendlyName} !~ /$self->{option_results}->{filter_name}/);
 
         my $duration = $options{custom}->convert_duration(time_string => $job->{properties}->{duration});
+        my $end_time = $options{custom}->convert_iso8601_to_epoch(time_string => $job->{properties}->{endTime});
+
+        my $ts_timeframe = time() - $self->{timeframe};
+        next if ($ts_timeframe > $end_time);
```

Comment:
Once option is locally defined, make a filter similar to the one line 160 would avoid changing the behavior of all already deployed check _behind the scene_